numpy/__init__.pyi

```diff
@@ -382,38 +382,38 @@ class _real_generic(generic):  # type: ignore
 class number(generic): ...  # type: ignore
 
 class bool_(_real_generic):
-    def __init__(self, value: object = ...) -> None: ...
+    def __init__(self, __value: object = ...) -> None: ...
 
 class object_(generic):
-    def __init__(self, value: object = ...) -> None: ...
+    def __init__(self, __value: object = ...) -> None: ...
 
 class datetime64:
     @overload
     def __init__(
-        self, _data: Union[datetime64, str, dt.datetime] = ..., _format: str = ...
+        self, __value: Union[datetime64, str, dt.datetime] = ..., __format: str = ...
```

Comment:
Is it possible to use positional-only argument syntax here instead, as

```suggestion
        self,value: Union[datetime64, str, dt.datetime] = ..., format: str = ..., /
```

Reply:
> I can't tell if this syntax is allowed in pyi files prior to python 3.8,

Unfortunately it isn't; mypy and the likes will start raising all sorts of alarm bells with such syntaxes.